### DISCUSSION TITLE
Looks like 18.5 (release) works fine

### DISCUSSION THREAD

**Discussion by fat-tire** [fat-tire/resolve#35] [General]
Some changes were made to support the betas a few weeks back, but now that it's out, I just confirmed that the `main` branch seems to build great with 18.5 and runs without any new issues.

Notice anything wrong?  Post here or file an issue.
ft

**Comment by litozpl** [comment]
Hi, yes it's work for me ;)
DA 18.5b on Ubuntu 23.04
